addons/crm/models/crm_lead.py

```diff
@@ -1107,13 +1107,10 @@ def _merge_opportunity_history(self, opportunities):
         self.ensure_one()
         for opportunity in opportunities:
             for message in opportunity.message_ids:
+                subject = message.subject and _("From %s : %s") % (opportunity.name, message.subject) or opportunity.name
```

Comment:
And I don't think you are converting it correctly.
The `From %(source_name)s` should stay there no matter what. It's only the `: %(source_subject)s` part that should not be "False" is if is not set.